src/app/shared/models/mock/mock-issue.ts

```
@@ -0,0 +1,22 @@
+import { Issue } from '../issue';
+
+export const mockIssue: Issue = {
```

Should we also mock responses for failed requests?